samples/skills/QASkill/ContextQuery/skprompt.txt

```diff
@@ -17,27 +17,27 @@ essay: longer answers. You can have sub-elements such as fact and fiction
 TODAY is {{time.Date}}
 FIRST NAME: {{$firstname}}
 LAST NAME: {{$lastname}}
-CITYl {{$city}}
+CITY: {{$city}}
 STATE: {{$state}}
 COUNTRY: {{$country}}
-{{recall $input}}
+{{memory.recall $input}}
```

Comment:
in all our demos we've always assumed there's a global "recall" function, which is a nice to have. Why this change?

Reply:
I liked the consistency with `time` above but will revert.